jobs/build/github-ldap-mapping-update/scripts/github-ldap-mapping-update.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -x
+
+ldapsearch -LLL -x -h ldap.corp.redhat.com -b ou=users,dc=redhat,dc=com '(rhatSocialURL=GitHub*)' rhatSocialURL uid 2>&1 | tee /tmp/out
+ldap-users-from-github-owners-files --ldap-file /tmp/out --mapping-file /tmp/mapping.yaml
```

Comment:
What is the path to `ldap-users-from-github-owners-files`?

Reply:
It is available in $PATH

Reply:
This one is good to go then.